src/components/tooltip/tooltip.tsx

```diff
@@ -11,16 +12,25 @@ interface TooltipProps {
     className?: string;
     open?: boolean;
     onClose?: () => void;
+    strategy?: "fixed" | "absolute" | "portal";
 }
 
-export const Tooltip = ({ children, title, position = "right", className, open }: TooltipProps) => {
+export const Tooltip = ({
+    children,
+    title,
+    position = "right",
+    className,
+    open,
+    strategy = "absolute",
+}: TooltipProps) => {
     const [referenceElement, setReferenceElement] = useState<HTMLDivElement>();
     const [popperElement, setPopperElement] = useState<HTMLDivElement>();
     const [show, setShow] = useState(false);
     const [isControlled, setIsControlled] = useState(false);
 
     const { styles, attributes } = usePopper(referenceElement, popperElement, {
         placement: position,
+        strategy: strategy !== "portal" ? strategy : "absolute",
```

Comment:
This means that if the strategy is "portal", we're passing "absolute" to the popper strategy, right?

Maybe it would be more obvious to do `strategy: strategy === "fixed" ? "fixed" : "absolute",`?

Reply:
Makes sense, done!